### PR TITLE
added note for loadbalancer services using GCP

### LINKED ISSUE #8745: [enterprise-3.9] Edit suggested in file install_config/configuring_gce.adoc
### Which section(s) is the issue in?
Configuring GCE

### What needs fixing?
There are some ansible variables that can be used, for instance [openshift_gcp_multizone](https://github.com/openshift/openshift-ansible/blob/release-3.9/roles/openshift_cloud_provider/tasks/gce.yml#L33) can be used and there is a [PR](https://github.com/openshift/openshift-ansible/pull/7980/files) for `openshift_gcp_node_tags`

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Fixes #8745

### CODE REVIEW

File: install_config/configuring_gce.adoc [hunk 1]
```diff
@@ -48,8 +48,9 @@ During xref:../install_config/install/advanced_install.adoc#install-config-insta
 
 [IMPORTANT]
 ====
-When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
+* When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
 parameters must be defined.
+* For running loadbalancer services using the Google compute platform, the nodes (Compute Engine VM instances) must be tagged with `<openshift_gcp_prefix>ocp`. That is, `prefix` with the characters `ocp` at the end. For example, if the value of parameter `openshift_gcp_prefix` is set to `mycluster`, the nodes must be tagged with `myclusterocp`. See link:https://cloud.google.com/vpc/docs/add-remove-network-tags[Adding and Removing Network Tags] for more information on how to add network tags to Compute Engine VM instances.
```
Comment (e-minguez) [inline install_config/configuring_gce.adoc]: I'd say:
"Add the `ocp` suffix" instead prefix at the end.

**Review by e-minguez — COMMENTED** [review]
Minimum change, otherwise LGTM

File: install_config/configuring_gce.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply install_config/configuring_gce.adoc]: yup 👍

**Comment by gaurav-nelson** [comment]
@openshift/team-documentation PTAL

File: install_config/configuring_gce.adoc [hunk 2]
```diff
@@ -48,8 +48,9 @@ During xref:../install_config/install/advanced_install.adoc#install-config-insta
 
 [IMPORTANT]
 ====
-When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
+* When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
 parameters must be defined.
+* For running loadbalancer services using the Google compute platform, the nodes (Compute Engine VM instances) must be tagged with `<openshift_gcp_prefix>ocp` (add `ocp` suffix). For example, if the value of parameter `openshift_gcp_prefix` is set to `mycluster`, the nodes must be tagged with `myclusterocp`. See link:https://cloud.google.com/vpc/docs/add-remove-network-tags[Adding and Removing Network Tags] for more information on how to add network tags to Compute Engine VM instances.
```
Comment (mburke5678) [inline install_config/configuring_gce.adoc]: @gaurav-nelson 
s/parameter `openshift_gcp_prefix`/the `openshift_gcp_prefix` parameter

File: install_config/configuring_gce.adoc [hunk 3]
```diff
@@ -48,8 +48,15 @@ During xref:../install_config/install/advanced_install.adoc#install-config-insta
 
 [IMPORTANT]
 ====
-When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
+* When using GCE, the `openshift_gcp_project` and `openshift_gcp_prefix`
 parameters must be defined.
+* For running loadbalancer services using the Google compute platform, the nodes
```
Comment (bfallonf) [inline install_config/configuring_gce.adoc]: @gaurav-nelson From a quick search of the docs, I think s/loadbalancer/load balancer

Unless you're talking about the LoadBalancer service?

**Pull request merged**